enterprise/changelog.md

```diff
@@ -12,6 +12,7 @@ New releases are announced on the [reviewable-enterprise-announce mailing list](
 - Fix: adapt to new GitHub OAuth token format.  This fixes "Unable to decrypt token with any key" errors.  You do _not_ need to change or fix your token encryption private key.
 - Fix: reduce Docker image size back <90MB.  The previous release accidentally bloated it a bit.
 - Fix: ensure that critical file diffs are proposed, even if the prior revision was modified to base. [See issue #342](https://github.com/Reviewable/Reviewable/issues/342)
+- New: display a revision cell with both an `action` and a `prior action` when rebased revisions are matched using heuristics. See the [docs](https://docs.reviewable.io/files.html#revision-cells) for details.
```

Comment:
I'd say this is more of an `Upd`.  Also, note that within each version I sort entries as `New` -> `Upd` -> `Fix`. 

I'd reword this to be more end-user-friendly, since `action` and `prior action` are purely internal terms:  `display an inner status color in rebased revision cells relative to the matched revision, which might not be the preceding one.  See the [docs]() for details.`

Should also have a separate entry about the reverted marker.